### PR TITLE
RAC-1042: Add step count in job execution table

### PR DESCRIPTION
**Pull request by ValentinMumble**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: upgrades/schema/Version_6_0_20211122154203_add_step_count_in_job_execution.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+final class Version_6_0_20211122154203_add_step_count_in_job_execution extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->skipIf(
+            $schema->getTable('akeneo_batch_job_execution')->hasColumn('step_count'),
+            'step_count column already exists in akeneo_batch_job_execution'
+        );
+
+        $this->addSql("ALTER TABLE akeneo_batch_job_execution ADD step_count INT NULL");
```
Comment (StevenVAIDIE) [inline upgrades/schema/Version_6_0_20211122154203_add_step_count_in_job_execution.php:19]: We didn't update the previous step_count ? Maybe we didn't need to do it for is_stoppable ?

File: upgrades/test_schema/Version_6_0_20211119154203_add_is_stoppable_in_job_execution_Integration.php [hunk 2]
```diff
@@ -38,7 +33,7 @@ protected function setUp(): void
         $this->connection = $this->get('database_connection');
     }
 
-    public function test_it_adds_a_new_type_column_to_the_connection_table(): void
```
Comment (StevenVAIDIE) [inline upgrades/test_schema/Version_6_0_20211119154203_add_is_stoppable_in_job_execution_Integration.php:41]: :eyes:

File: upgrades/test_schema/Version_6_0_20211122154203_add_step_count_in_job_execution_Integration.php [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    {
+        parent::setUp();
+        $this->connection = $this->get('database_connection');
+    }
+
+    public function test_it_adds_step_count_column_to_the_job_execution_table(): void
+    {
+        $this->dropColumnIfExists();
+
+        $stoppableJobInstanceId = $this->createJobInstance('csv_user_group_export');
```
Comment (StevenVAIDIE) [inline upgrades/test_schema/Version_6_0_20211122154203_add_step_count_in_job_execution_Integration.php:40]:
```suggestion
        $jobInstanceId = $this->createJobInstance('csv_user_group_export');
```

**Review by StevenVAIDIE — APPROVED** [review]

**Review by StevenVAIDIE — COMMENTED** [review]
Missing the EE PR for the schema (like this PR [src/Akeneo/Tool/Bundle/DatabaseMetadataBundle/Resources/reference.pimdbschema.txt](https://github.com/akeneo/pim-enterprise-dev/pull/12353/files#diff-bd537553805ad68be9cfcb5b6428010c9edf4ec8b03bf33d3d0c1f153eb72ae2R214))

**Review by pierrejolly — APPROVED** [review]

**Pull request merged**